libraries/AP_Compass/AP_Compass.h

```diff
@@ -38,9 +38,9 @@
 
 // define default compass calibration fitness and consistency checks
 #define AP_COMPASS_CALIBRATION_FITNESS_DEFAULT 16.0f
-#define AP_COMPASS_MAX_XYZ_ANG_DIFF radians(90.0f)
-#define AP_COMPASS_MAX_XY_ANG_DIFF radians(60.0f)
-#define AP_COMPASS_MAX_XY_LENGTH_DIFF 200.0f
+#define AP_COMPASS_MAX_XYZ_ANG_DIFF radians(30.0f)
+#define AP_COMPASS_MAX_XY_ANG_DIFF radians(20.0f)
+#define AP_COMPASS_MAX_XY_LENGTH_DIFF 100.0f
```

Comment:
The first two values are 1/3 of the original value, but this one (`AP_COMPASS_MAX_XY_LENGTH_DIFF`) is only 1/2. Is this intentional or is there no correlation to the previous two values?

Reply:
This is intentional. The first two are angles, the last one is magnitude. Different dimensional planes